Explorer/Assets/DCL/AvatarRendering/Emotes/Components/CharacterEmoteComponent.cs

```diff
@@ -9,7 +10,18 @@ public struct CharacterEmoteComponent
         public bool EmoteLoop;
         public EmoteReferences? CurrentEmoteReference;
         public int CurrentAnimationTag;
+        // TODO: Replace StopEmote with component StopEmoteIntent
         public bool StopEmote;
```

Comment:
By the way, why do we like to do it like that? Why use components instead of flags? In my mind, that's a bad idea because every time you add or remove a component, the whole instance gets copied to another archetype. So what would normally be a write of 4 bytes is an arbitrarily large memory copy.